client/src/components/card/PreviewCard.jsx

```diff
@@ -4,11 +4,19 @@ import { useRef, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import {
   addContentToWatchHistory,
+  addContentToWatchList,
+  deleteContentFromWatchList,
```

Comment:
I believe remove would be better than delete

Reply:
Done. Thanks!